calculatorproject.js

```diff
@@ -1 +1,5 @@
-calculatorproject.js
\ No newline at end of file
+function addNumberToDisplay(number) {
+  // Function Implementation...
+}
+
+displayScreen.value = number;
```

Comment:
@evanhearne the function is defined well here -> this line displayScreen.value = number; is outside the function however so wont be called. The commented line `// Function Implementation...` is there to guide you where to put the code that you want executed when the function is called.

Reply:
OK, thank you. Will make changes now.